shared/utils/math.ts

```diff
@@ -10,6 +10,15 @@ export const math = {
         return a < max ? a > min ? a : min : max;
     },
 
+    /**
+     * Generate a random floating-point value.
+     * @param min The minimum value that can be generated.
+     * @param max The maximum value that can be generated.
+    */
+    randomFloat(min: number, max: number): number {
```

Comment:
we already have a random float function on util.ts
https://github.com/leia-uwu/resurviv/blob/master/shared/utils/util.ts#L36